graylog2-web-interface/src/pages/ClusterConfigurationPage.tsx

```diff
@@ -41,7 +43,7 @@ const ClusterConfigurationPage = () => {
       <div>
         <PageHeader title="Cluster Configuration">
           <span>
-            This page provides a real-time overview of the nodes in your Graylog cluster. You can pause message
+            This page provides a real-time overview of the nodes in your {productName} cluster. You can pause message
```

Comment:
We can also leave out the product name here, I would say.

Reply:
Do you prefer this change for all cases where we currently use `{productName} cluster`?

Reply:
I adjusted it as suggested in similar cases as well.